src/main/resources/static/views/interest/InvestorContact.html

```diff
@@ -0,0 +1,32 @@
+<div class="container" ng-init="getContact()">
+    <div class="row">
+        <div class="col-xs-12 col-sm-10 col-md-8 col-lg-8 col-lg-offset-2" style="margin-top: 50px;">
+            <div class="panel-card panel-success-card">
+                <div class="text-center"><h3>INVESTMENT INTEREST</h3></div>
+                <div class="panel-body">
+                    <div class="row">
+                        <div class="col-xs-12 text-center">
+                            <h4><b>Our goal:</b></h4>

```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this <b> tag by <strong>. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=Web%3ABoldAndItalicTagsCheck)